packages/hoppscotch-backend/src/team-invitation/team-invitation.service.ts

```diff
@@ -63,6 +66,32 @@ export class TeamInvitationService {
     );
   }
 
+  /**
+   * Get the team invite for an invitee with email and teamID.
+   * @param inviteeEmail invitee email
+   * @param teamID team id
+   * @returns an Either of team invitation for the invitee or error
+   */
+  async getTeamInviteByEmailAndTeamID(inviteeEmail: string, teamID: string) {
+    const isEmailValid = validateEmail(inviteeEmail);
+    if (!isEmailValid) return E.left(INVALID_EMAIL);
```

Comment:
This callee `getTeamInviteByEmailAndTeamID` function validates the new email.
The caller `AdminService.addUserToTeam` function also validates the same email.

So we can keep one and eliminate the other one.

Reply:
`getTeamInviteByEmailAndTeamID` is a service method inside team invitation service written with the motivation so that we could use it later for similar operation if required I think we can keep it.